screwdriver.yaml

```diff
@@ -5,6 +5,7 @@ jobs:
     main:
         environment:
             SD_SONAR_OPTS: "-Dsonar.sources=index.js,lib -Dsonar.tests=test -Dsonar.javascript.lcov.reportPaths=artifacts/coverage/lcov.info -Dsonar.testExecutionReportPaths=artifacts/report/test.xml"
+            NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
seems to be a sharp jump from 500MB default. Also sonar-scanner is a java based binary, don't this setting is going to affect it's performance

Reply:
from what I recall, more CPU to build seems to improve it's performance 🤔

Reply:
updated